content/release-notes/2023-07-25-storage-and-compute.md

```diff
@@ -0,0 +1,19 @@
+---
+label: 'Storage'
+---
+
+### What's new
+
+**Faster cold starts in all regions**
+
+Neon's _Auto-suspend_ feature scales a compute instance to zero after a period of inactivity. A characteristic of this feature is a "cold start", which occurs when an idle compute is restarted to process requests. Recently, cold-start times have been improved through a variety of enhancements, outlined below:
+
+- **Compute pools**: Instead of starting computes from zero, requests for computes are now served from pools of pre-started compute instances.
+- **Compute configuration optimization**: Configuration changes at compute startup were eliminated where possible.
+- **Caching of internal IP addresses**: Internal IP addresses are now cached to avoid waits for internal DNS routing.
+- **Concurrency improvements**: Concurrency optimizations were applied to the compute startup process.
+- **Code path optimization**: Code paths frequently accessed during compute startup were optimized.
+
+With these improvements, cold starts are faster in all [supported regions](/docs/introduction/regions). Cold starts in the `US East (Ohio) — aws-us-east-2` region, where the Neon Control Plane is hosted, are the fastest at approximately 500ms. Work is underway to deploy the Neon Control Plane regionally to enable the same millisecond startup times for projects in all regions. For more information about this effort, please check out the blog post from the Neon engineering team: [Cold starts just got hot](https://neon.tech/blog/cold-starts-just-got-hot).
```

Comment:
I noticed two options for writing `500 ms` and `500ms` (with and without space). Should we follow the same style?

Reply:
Thank you. Should have a space. Will use 500 ms.